src/components/oculus-touch-controls.js

```diff
@@ -81,6 +88,13 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
     this.previousButtonValues = {};
     this.bindMethods();
     this.isControllerPresent = isControllerPresent; // to allow mock
+    this.getGamepadsByPrefix = getGamepadsByPrefix; // to allow mock
+  },
+
+  update: function (oldData) {
+    if ((oldData.emulated !== undefined) && (this.data.emulated !== oldData.emulated)) {
+      this.checkIfControllersPresent();
```

Comment:
the original reason for that was to avoid flagging the first update call (right after init) as needing the controllers check, since play() will do it.